lte/cloud/go/services/policydb/obsidian/models/conversion.go

```diff
@@ -314,3 +367,69 @@ func (m *NetworkSubscriberConfig) GetFromNetwork(network configurator.Network) i
 func (m *NetworkSubscriberConfig) ToUpdateCriteria(network configurator.Network) (configurator.NetworkUpdateCriteria, error) {
 	return orc8rModels.GetNetworkConfigUpdateCriteria(network.ID, lte.NetworkSubscriberConfigType, m), nil
 }
+
+func (m *PolicyQosProfile) FromBackendModels(networkID string, key string) error {
+	config, err := configurator.LoadEntityConfig(networkID, lte.PolicyQoSProfileEntityType, key)
+	if err != nil {
+		return err
+	}
+	*m = *config.(*PolicyQosProfile)
+	return nil
+}
+
+func (m *PolicyQosProfile) ToUpdateCriteria(networkID string, key string) ([]configurator.EntityUpdateCriteria, error) {
+	if key != m.ID {
+		return nil, errors.New("id field is read-only")
+	}
+
+	exists, err := configurator.DoesEntityExist(networkID, lte.PolicyQoSProfileEntityType, key)
```

Comment:
this check belongs to the caller. this method signature should not include an error

Reply:
In general I agree, but the method is an iface method of `PartialEntityModel`, and for those impls this is a common pattern